sdks/go/pkg/beam/runners/dataflow/dataflow.go

```diff
@@ -71,7 +72,6 @@ var (
 	tempLocation           = flag.String("temp_location", "", "Temp location (optional)")
 	machineType            = flag.String("worker_machine_type", "", "GCE machine type (optional)")
 	minCPUPlatform         = flag.String("min_cpu_platform", "", "GCE minimum cpu platform (optional)")
-	workerJar              = flag.String("dataflow_worker_jar", "", "Dataflow worker jar (optional)")
```

Comment:
This is technically breaking even if it previously no-oped (now it will fail) - could we call it out in the release notes as well?

Reply:
I'll add it back, and mark as deprecated, not worth breaking people over it.